ZelBack/src/services/dockerService.js

```diff
@@ -583,6 +585,16 @@ async function appDockerCreate(appSpecifications, appName, isComponent, fullAppS
     },
   };
 
+  // get docker info about Backing Filesystem
+  // eslint-disable-next-line no-use-before-define
+  const dockerInfoResp = await dockerInfo();
+  log.info(dockerInfoResp);
+  const driverStatus = dockerInfoResp.DriverStatus;
+  const backingFs = driverStatus.find((status) => status[0] === 'Backing Filesystem'); // d_type must be true for overlay, docker would not work if not
+  if (backingFs && backingFs[1] === 'xfs') {
```

Comment:
missing mount quota check

Reply:
Can you explain more? It must have the 'pquota' mount option right?
Does the dockerInfo output that?

Reply:
dockerInfo dont have such option i created custom function base on df and mount .
Example belown:
```
const cmd = require('node-cmd');

async function getDfDevice(directory) {
  return new Promise((resolve, reject) => {
    cmd.get(`df --output=source,target ${directory}`, (error, data, stderr) => {
      if (error) {
        resolve(false);
        return;
      }
      const lines = data.trim().split('\n');
      for (let i = 1; i < lines.length; i++) {
        const columns = lines[i].split(/\s+/);
        if (columns[0] !== "") {
          resolve(columns[0]);
          return;
        }
      }
      resolve(false);
    });
  });
}

async function hasQuotaOptionForDevice(device) {
  return new Promise((resolve, reject) => {
    cmd.get(`mount | grep "${device}"`, (error, data, stderr) => {
      if (error) {
        resolve(false);
        return;
      }
      const mountInfo = data.trim();
      if (!mountInfo) {
        resolve(false);
        return;
      }
      if (mountInfo.includes('quota') && mountInfo.includes('xfs')) {
        resolve(true);
      } else {
        resolve(false);
      }
    });
  });
}

async function main() {
  const directoryToCheck = '/var/lib/docker';
  try {
    const device = await getDfDevice(directoryToCheck);
    if (device) {
      if (await hasQuotaOptionForDevice(device)) {
        console.log('OK');
      }
    }
  } catch (error) {
    console.error('Error:', error);
  }
}

main();
```

Reply:
i updated commented for pr with mount info

Reply:
Cool, I will include that there then. 
What are like the defaults for when we install OS on XFS? Is quota a standard?

Reply:
Cool your code fix commited :)